data/json/items/melee.json

```diff
@@ -1696,7 +1696,7 @@
     "price": 1050,
     "price_postapoc": 300,
     "material": [ "aluminum", "wood" ],
-    "flags": "NONCONDUCTIVE",
+    "flags": [ "NONCONDUCTIVE", "FRAGILE_MELEE" ],
```

Comment:
I'm not particularly familiar with cheap replica axes, but as long as the haft extends through the head and it's not just welded on, I dont see fragile_melee being justified.